src/__tests__/tigris.rpc.spec.ts

```diff
@@ -571,6 +572,48 @@ describe("rpc tests", () => {
 			}
 		});
 	});
+
+	it ("publish", () => {
+		const tigris = new Tigris({serverUrl: "0.0.0.0:" + SERVER_PORT, insecureChannel: true});
+		const db = tigris.getDatabase("test_db");
+		const topic = db.getTopic<Alert>("test_topic");
+		expect(topic.topicName).toBe("test_topic");
+
+		const promise = topic.publish({
+			id: 34,
+			text: "test"
+		});
+
+		promise.then(alert => {
+			expect(alert.id).toBe(34);
+			expect(alert.text).toBe("test");
+		});
+
+		return promise;
+	});
+
+	it ("subscribe", (done) => {
+		const tigris = new Tigris({serverUrl: "0.0.0.0:" + SERVER_PORT, insecureChannel: true});
+		const db = tigris.getDatabase("test_db");
+		const topic = db.getTopic<Alert>("test_topic");
+		let success = true;
+
+		topic.subscribe({
+			onNext(alert: Alert) {
+				expect(alert.id).toBe(34);
+				expect(alert.text).toBe("test");
+				expect(success).toBe(true);
+				done();
+			},
+			onEnd() {
+				// not expected to be called
```

Comment:
should we put fail() here to get tighter grip?

Reply:
I've tried to add this before, it hits this bug: https://github.com/facebook/jest/issues/11698